packages/noya-state/src/reducers/canvasReducer.ts

```diff
@@ -593,6 +636,46 @@ export function canvasReducer(
 
             break;
           }
+          case 'maybeMoveGradientElipseLength': {
```

Comment:
In general the `maybe` states do 2 things:

1. Let us take a snapshot of the page data so we can always do calculations with the state at the start of the interaction. This is important since sometimes interactions can temporarily have an invalid state (e.g. scaling a layer to a width of 0)
2. Enforce a dragging threshold of a few pixels, so the user doesn't accidentally adjust things by a tiny amount

We don't really need (1) in this case, since there's no "invalid" ellipse length. However it's still useful for (2), so I think we should keep it.

We can delete this empty switch statement though.